contributors/design-proposals/api-machinery/admission-control-webhooks.md

```diff
@@ -167,31 +158,33 @@ InitializerConfiguration and ExternalAdmissionHookConfiguration.
 
 InitializerConfiguration will not join `admissionregistration.k8s.io/v1beta1` at this time.
 
-Any webhooks that register with v1alpha1 may or may not be surprised when they start getting Versioned data.  But we don't make any promises for Alpha, and detecting which API was used to register is too complex.
+Any webhooks that register with v1alpha1 may or may not be surprised when they start getting versioned data.  But we don't make any promises for Alpha, and this is a very important bug to fix.
 
 
 ## Order of Admission 
 
-The recommended order on kubernetes.io for `--admission-control` will be: `MutatingAdmissionWebhook,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,ValidatingAdmissionWebhook,ResourceQuota`
-
- This order has the following properties:
-
+At kubernetes.io, we will document the ordering requirements or just recommend a particular order for `--admission-control`. A starting point might be `MutatingAdmissionWebhook,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,ValidatingAdmissionWebhook,ResourceQuota`.
```

Comment:
I'm ok with this wording.  Before we open a pull setting this, you should take a look at how the generic apiserver allows you to specify a fully ordered set of all plugins with specific ones listed as "default to off".  This allows users to do the more common "I want to enforce X" without trying to own the complete list of all admission plugins.  It still allows users to override the list and say "I want exactly these in this order", but that use-case is less common.